pom.xml

```diff
@@ -20,6 +20,22 @@
         </plugins>
     </build>
     <dependencies>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>3.141.59</version>
```

Comment:
When working on a team, always ask your team for the versions of new dependencies